src/elevenlabs/textToSpeechFile.ts.ts

```diff
@@ -7,15 +7,19 @@ import * as dotenv from 'dotenv';
 
 dotenv.config();
 
-const ELEVENLABS_API_KEY = process.env.ELEVENLABS_API_KEY;
-
-const client = new ElevenLabsClient({
-  apiKey: ELEVENLABS_API_KEY,
-});
-
 export const createAudioFileFromText = async (
   text: string,
 ): Promise<string> => {
+  const ELEVENLABS_API_KEY = process.env.ELEVENLABS_API_KEY;
```

Comment:
why not to use from config?

Reply:
yes, we already migrate everthing to use from `config`